rust_src/remacs-sys/lib.rs

```diff
@@ -124,6 +124,17 @@ pub enum PseudovecType {
     PVEC_FONT, /* Should be last because it's used for range checking.  */
 }
 
+#[repr(isize)]
```

Comment:
Is this really correct for a C enum? I thought the default type there was a c_int.

Better use `#[repr(C)]`?